src/deluge/modulation/arpeggiator.h

```diff
@@ -28,7 +28,9 @@
 class PostArpTriggerable;
 class ParamManagerForTimeline;
 
-#define SPREAD_LOCK_MAX_SAVED_VALUES 16
+#define RANDOMIZER_LOCK_MAX_SAVED_VALUES 16
+#define ARP_NOTE_NONE 32767
+#define ARP_MAX_INSTRUCTION_NOTES 4
```

Comment:
Can these be `constexpr uint32_t` values instead?

Reply:
Actually the `*MAX*` items should be `size_t` rather than `uint32_t` (it's the same on our platform but one day the difference will be important).